recipe/meta.yaml

```diff
@@ -11,16 +11,17 @@ source:
 
 build:
     skip: True  # [win]
-    number: 4
+    number: 5
 
 requirements:
     build:
-        - zlib 1.2.*
+        - zlib
+        - gnuconfig  # [unix]
         - {{ compiler('c') }}
         - make
         - flex
     run:
-        - zlib 1.2.*
+        - zlib
```

Comment:
The pinning seems to have been added 6 years ago, before global pinnings were introduced.